pennylane/devices/default_tensor.py

```diff
@@ -351,6 +475,50 @@ def _initial_mps(self, wires: qml.wires.Wires) -> "qtn.MatrixProductState":
             tags=[str(l) for l in wires.labels] if wires else None,
         )
 
+    def draw(self, color="auto", **draw_opts):
+        """
+        Draw the current quantum circuit using `quimb`'s functionality.
+
+        Internally, it uses `quimb`'s `draw` method.
+
+        Args:
+            color (str): The color of the tensor network diagram. Default is ``"auto"``.
+            **draw_opts: Additional keyword arguments for the ``quimb``'s ``draw`` function. For more information, see the
+                `quimb's draw documentation <https://quimb.readthedocs.io/en/latest/tensor-drawing.html>`_.
```

Comment:
An example would be great! (no need to plot the output in the docs, but just for people to have something to run)